controller/admin_message/manager.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, List
+from datetime import datetime
+from submodules.model.business_objects import admin_message
+from submodules.model.models import AdminMessage
+from util.notification import send_global_update_for_all_organizations
+
+
+def get_all_admin_messages() -> List[AdminMessage]:
+    get_all_active_admin_messages()  # set messages on archived if they should be archived
```

Comment:
imo not optimal regarding naming & function since we dont use the return value. Maybe another (private) function to check/set archived which is used in both (and returns True/False if something was archived)

Reply:
I understand where you are coming from but I had this specific in mind building this, I renamed the method since the naming really is poor. Lets talk about this and then decide if it should be changed :)